build/nested-cluster/BUILD

```diff
@@ -0,0 +1,65 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build", "docker_bundle")
+
+filegroup(
+    name = "test-targets",
+    srcs = [
+        "//test/e2e:e2e.test",
+        "//vendor/github.com/onsi/ginkgo/ginkgo",
+    ],
+)
+
+docker_build(
+    name = "nested-kubelet-build",
+    base = "@nested-kubelet-base//image",
+    debs = [
+        "//build/debs:kubelet",
+        "//build/debs:kubectl",
+        "//build/debs:kubeadm",
+    ],
+    entrypoint = ["/init-wrapper.sh"],
+    files = [
+        "init-wrapper.sh",
+        "integ-test.sh",
+        "start.sh",
+        "//build:master-components-amd64",
+        "//build/nested-cluster/util:k8sversion",
```

Comment:
even easier: use the genrule `//build:version`, which produces a file named `version` containing the "git version" string.